content/concept-amendments.md

```diff
@@ -1,6 +1,6 @@
 # Amendments #
 
-_(New in [version 0.31.0](https://wiki.ripple.com/Rippled-0.31.0))_
+_(Introduced in [version 0.31.0](https://wiki.ripple.com/Rippled-0.31.0))_
```

Comment:
Just a suggestion, should we consider using badges for this? For example:

# Amendments #
![Introduced in version 0.31.0](https://img.shields.io/badge/Version-0.31.0-blue.svg?style=flat)

This lets us do other fancy things. For example:

# FlowV2 #
![Vetoed](https://img.shields.io/badge/Status-Vetoed-red.svg?style=flat)

# MultiSign #
![Ratified](https://img.shields.io/badge/Status-Ratified-greed.svg?style=flat)